### PR TITLE
SDNTB-135 change untranslated strings in SD

### PR DESCRIPTION
**Pull request by vladnicoara**
Conflicts:
	scripts/superdesk-authoring/views/article-edit.html
	scripts/superdesk-authoring/views/send-item.html

### CODE REVIEW

**Comment by akintolga** [comment]
:+1:

**Comment by mdhaman** [comment]
:+1:

File: scripts/superdesk-authoring/views/article-edit.html [hunk 1]
```diff
@@ -38,8 +38,7 @@
     </div>
 </div>
 
-<div ng-if="schema.feature_image" sd-validation-error="error.feature_image" order="{{editor.feature_image.order}}"
-    data-required="schema.feature_image.required" class="field featured-image">
+<div ng-if="schema.feature_image" sd-validation-error="error.feature_image" data-required="schema.feature_image.required" class="field featured-image">
```
Comment (ioanpocol) [inline scripts/superdesk-authoring/views/article-edit.html]: Why order is deleted?

File: scripts/superdesk-authoring/views/article-edit.html [hunk 2]
```diff
@@ -156,8 +155,9 @@
   <div ng-if="item.type == 'picture'" class="full-preview" sd-ratio-calc>
       <div>
           <span translate>Original</span>
-          <div sd-item-rendition data-item="item" data-rendition="baseImage" ng-click="preview(item);"></div>
+          <div sd-item-rendition data-item="item" data-rendition="baseImage"></div>
```
Comment (ioanpocol) [inline scripts/superdesk-authoring/views/article-edit.html]: Why preview is deleted?

File: scripts/superdesk-authoring/views/article-edit.html [hunk 3]
```diff
@@ -241,4 +241,4 @@
          data-onsearch="searchSignOff(search)"
          data-display-user="{{ getSignOffMapping() }}"
     ></div>
-</div>
+</div>
```
Comment (ioanpocol) [inline scripts/superdesk-authoring/views/article-edit.html]: Missing new line

File: scripts/superdesk/editor2/editor.js [hunk 4]
```diff
@@ -941,15 +941,55 @@ angular.module('superdesk.editor2', [
...
+                            tagNames: ['b'],
+                            contentDefault: '<b>' + gettextCatalog.getString('B') + '</b>'
+                        });
+                        break;
+                    case 'underline':
+                        editorConfig.toolbar.buttons.push({
+                            name: 'underline',
+                            action: 'underline',
+                            aria: gettextCatalog.getString('underline'),
+                            tagNames: ['i']
```
Comment (ioanpocol) [inline scripts/superdesk/editor2/editor.js]: Tag name u instead i.

**Comment by ioanpocol** [comment]
:+1:  (except minor comments)

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**